src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -968,8 +965,8 @@ def _validate_vmss_create_load_balancer_or_app_gateway(cmd, namespace):
     if namespace.load_balancer is None and namespace.application_gateway is None:
         if std_lb_is_available:
             balancer_type = 'loadBalancer'
-        else:
-            balancer_type = 'loadBalancer' if namespace.single_placement_group else 'applicationGateway'
+        else:  # mainly for Azure stack users
```

Comment:
Might be better to make this comment stronger. 
`# needed for Stack profile 2017_03_09`